kong.conf.default

```diff
@@ -1532,6 +1532,97 @@
                                  # same name/type will be synchronized to a
                                  # single query.
 
+#------------------------------------------------------------------------------
+# New DNS RESOLVER
+#------------------------------------------------------------------------------
+
+# This DNS resolver introduces global caching for DNS records across workers,
+# significantly reducing the query load on DNS servers.
+#
+# It provides observable statistics, you can retrieve them through the Admin API
+# `/status/dns`.
+
+#legacy_dns_client = off         # Disable the new DNS resolver, using the
+                                 # original DNS resolver. See above `dns_xxx`
+                                 # options for the original DNS resolver.
+
+#resolver_address =              # Comma-separated list of nameservers, each
+                                 # entry in `ip[:port]` format to be used by
+                                 # Kong. If not specified, the nameservers in
+                                 # the local `resolv.conf` file will be used.
+                                 # Port defaults to 53 if omitted. Accepts
+                                 # both IPv4 and IPv6 addresses.
```

Comment:
```suggestion
                                 # both IPv4 and IPv6 addresses.
```

Could we provide an example of IPv6 address and IPv4 address?

Reply:
fixed

Reply:
```
                                 # both IPv4 and IPv6 addresses.
                                 #
                                 # Examples:
                                 #
                                 # ```
                                 # resolver_address = 8.8.8.8
                                 # resolver_address = 8.8.8.8, [::1]
                                 # resolver_address = 8.8.8.8:53, [::1]:53
```

Reply:
Looks good.